src/table/components/head/HeadCellMenu.tsx

```diff
@@ -30,16 +31,18 @@ const MenuItem = ({
   sortDirection: string;
   sortFromMenu(evt: React.MouseEvent, sortOrder: string): void;
   setOpen: any;
+  isInteractionEnabled: boolean;
   isCurrentColumnActive: boolean;
 }) => {
-  const isDisabled = isCurrentColumnActive && sortOrder === (sortDirection === 'asc' ? 'A' : 'D');
+  let isDisabled = false;
+  isDisabled = !isInteractionEnabled || (isCurrentColumnActive && sortOrder === sortDirection);
   return (
     <ListItem disablePadding>
       <ListItemButton
         disabled={isDisabled}
         className="sn-table-head-menu-item-button"
         onClick={(evt) => {
-          sortFromMenu(evt, sortOrder);
+          sortFromMenu(evt, sortOrder.charAt(0).toUpperCase());
```

Comment:
you are doing this in two places. If you revert it here it makes sense with the reset

Reply:
Fixed